alvr/client_openxr/src/lib.rs

```diff
@@ -258,23 +247,22 @@ pub fn entry_point() {
         };
         let core_context = Arc::new(ClientCoreContext::new(capabilities));
 
-        let interaction_context = Arc::new(interaction::initialize_interaction(
-            &xr_context,
+        let interaction_context = Arc::new(RwLock::new(InteractionContext::new(
+            xr_session.clone(),
             platform,
-            parsed_stream_config
-                .as_ref()
-                .map(|c| c.prefers_multimodal_input)
-                .unwrap_or(false),
-            parsed_stream_config
-                .as_ref()
-                .and_then(|c| c.face_sources_config.clone()),
-            parsed_stream_config
-                .as_ref()
-                .and_then(|c| c.body_sources_config.clone()),
-        ));
+            exts.other
+                .contains(&META_SIMULTANEOUS_HANDS_AND_CONTROLLERS_EXTENSION_NAME.to_owned()),
+        )));
+        interaction_context
+            .write()
+            .select_sources(InteractionSourcesConfig {
+                face_tracking: None,
+                body_tracking: None,
+                prefers_multimodal_input: true,
```

Comment:
don't really see the reason for this? multimodal in lobby is useless and if we can toggle at any time like I suggest then we can only enable it when the server communicated that it wants to and save some perf

Reply:
Actually this is used in another branch, i haven't made a PR yet. Basically we poll the detached controllers and show them in the lobby, with a gray color